client/src/components/App.js

```diff
@@ -0,0 +1,90 @@
+import './App.css';
+import { DeidentifiedNotesApi } from '../apis';
+import { DeidentifyRequestFromJSON } from '../models';
+import React from 'react';
+import { Configuration } from '../runtime';
+import { DeidentifiedText, deidentificationStates } from './DeidentifiedText';
+import { DeidentificationConfigForm } from './DeidentificationConfigForm';
+
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
Something for you to consider if you have the time to experiment. It's easier and less to write functional React component rather than class component (what you have here). It's easier and less because you don't have to deal with life-cycle hooks and state that don't always get you the expected value. Most of us write React in functional way now. There are many online tutorials on how to convert a class component to a functional component. Check them out if you're interested.